### DISCUSSION TITLE
Thanks @Sreehari521

### DISCUSSION THREAD

**Discussion by dolphin007x** [Sreehari521/harso-css#2] [General]
Thanks a lot for letting me be a collaborator on the repo @Sreehari521 . Highly appreciate it.

Thanks,
@Adatta1276

**Comment by Genius398** [comment]
:)

**Reply by Sreehari521** [reply]
> :)

Anant, how's this library? Will u contribute?

**Comment by Sreehari521** [comment]
> Thanks a lot for letting me be a collaborator on the repo @Sreehari521 . Highly appreciate it.
> 
> Thanks, @Adatta1276

Yeah, if u contribute then plz send it as pull requests, then i will build a new version of it and publish in https://www.npmjs.com/package/harso-css

**Reply by dolphin007x** [reply]
Okay

**Reply by dolphin007x** [reply]
> > Thanks a lot for letting me be a collaborator on the repo @Sreehari521 . Highly appreciate it.
> > 
> > Thanks, @Adatta1276
> 
> Yeah, if u contribute then plz send it as pull requests, then i will build a new version of it and publish in https://www.npmjs.com/package/harso-css

Wait you made a npmJS package ?

**Reply by Sreehari521** [reply]
> > > Thanks a lot for letting me be a collaborator on the repo @Sreehari521 . Highly appreciate it.
> > > Thanks, @Adatta1276
> > 
> > 
> > Yeah, if u contribute then plz send it as pull requests, then i will build a new version of it and publish in https://www.npmjs.com/package/harso-css
> 
> Wait you made a npmJS package ?

Yeah , download it: npm install harso-css

**Reply by Sreehari521** [reply]
> Okay

Have you seen my yt channel: https://www.youtube.com/channel/UCYJKtrmrcSbFlNABR_JFWuQ/videos

**Reply by dolphin007x** [reply]
yes

**Reply by N3RDIUM** [reply]
> > > > Thanks a lot for letting me be a collaborator on the repo @Sreehari521 . Highly appreciate it.
> 
> > > > Thanks, @Adatta1276
> 
> > > 
> 
> > > 
> 
> > > Yeah, if u contribute then plz send it as pull requests, then i will build a new version of it and publish in https://www.npmjs.com/package/harso-css
> 
> > 
> 
> > Wait you made a npmJS package ?
> 
> 
> 
> Yeah , download it: npm install harso-css

One question: How do we upload packages to npm?

**Reply by Sreehari521** [reply]
> > > > > Thanks a lot for letting me be a collaborator on the repo @Sreehari521 . Highly appreciate it.
> > 
> > 
> > > > > Thanks, @Adatta1276
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > Yeah, if u contribute then plz send it as pull requests, then i will build a new version of it and publish in https://www.npmjs.com/package/harso-css
> > 
> > 
> > > 
> > 
> > 
> > > Wait you made a npmJS package ?
> > 
> > 
> > Yeah , download it: npm install harso-css
> 
> One question: How do we upload packages to npm?

# i will tell, but plz don't do it to harso-css bcz I will only upload the new version after 100+ changes in this library, the way you can do is
## 1) Go to https://npmjs.com
## 2) Create an account
## 3) Paste this in your terminal: ``` npm login ```
## 4) After logged in, create a package.json like ours
## 5) If you want to contribute in any other library, get the code of that library, Eg: harso-css, then create a new version Eg) If we 3.0.4 the you create 3.0.5 like ```"version": "3.0.5"``` in the package.json and type ```npm publish``` in your Terminal!
## 6) Otherwise, set the version into 1.0.0 and type ```npm publish``` in your Terminal!

**Reply by N3RDIUM** [reply]
Thanks, @Sreehari521  !

**Comment by devashish2024** [comment]
Really, You're using a another Terminal Supports in ONLY 64-bit windows, I'm using 32-bit that are not much faster, not much safer and not much secure. in this there It doesn't works. sorry!

**Comment by devashish2024** [comment]
Really, You're using a another Terminal Supports in ONLY 64-bit windows, I'm using 32-bit that are not much faster, not much safer and not much secure. in this there It doesn't works. sorry! i am talking about npm and node js.
